x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/scalar/stringcontains/StringContains.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.eql.expression.function.scalar.stringcontains;
```

Comment:
Package I think should be `org.elasticsearch.xpack.eql.expression.function.scalar.string`.

Reply:
Moved to the scalar.string package.